cpp/src/Slice/Scanner.l

```diff
@@ -646,7 +622,7 @@ int checkIdentifier(string& id)
     {
         DefinitionContextPtr dc = unit->currentDefinitionContext();
         assert(dc);
-        if(dc->findMetaData("underscore") != "underscore") // no 'underscore' file metadata
+        if(dc->findMetaData("underscore") != "underscore") // no 'underscore' global metadata
```

Comment:
I believe 'file' is the correct term still. We moved away from 'global' some time ago.
```suggestion
        if(dc->findMetaData("underscore") != "underscore") // no 'underscore' file metadata
```